py/torch_tensorrt/dynamo/conversion/impl/elementwise/base.py

```diff
@@ -116,9 +121,9 @@ def convert_binary_elementwise(
     # tensor. This is safe because broadcast will pad dimensions on the left
     # (prepend) to make lhs and rhs shape compatible.
     if network.has_implicit_batch_dimension:
-        if isinstance(lhs_val, torch.Tensor):
```

Comment:
Can we remove the `network.has_implicit_batch_dimension` altogether here?

Reply:
Sure, I will make this change